### DISCUSSION TITLE
Missing icons/emojis in notifications

### DISCUSSION THREAD

**Discussion by formfarbefabs** [zepp-health/zeppos-forum#207] [User Feedback]
Hello, i honestly like my new GTS4 - but there seems to be a lack of emojis on the OS, when receiving notifications from e. g. messengers (Whatsapp, Signal...). And that makes the Amazfit product really look broken...

**Comment by liqin-ZeppHealth** [comment]
Hi @formfarbefabs , thanks for your feedback, better user experience is always the goal of Amazfit products. As this forum we are mainly addressing some technical issues from developers, so I suggest that you can submit your feedback on the Zepp APP, or you can send feedback to  support@amazfit.com , our colleagues will follow up as necessary. thank you
